templates/case/summary.html.twig

```diff
@@ -9,7 +9,7 @@
 {% set caseinfo = [
     {
         "label": "Created by",
-        "value": case.complainant
+        "value": case.createdBy
```

Comment:
Are we relying on `case.createdBy` being converted to a string (via `__toString()`) here?

Reply:
Yes, the `__toString()` method of the `User` entity.